webapp/packages/core-blocks/src/FormControls/InputFiles.tsx

```diff
@@ -195,28 +172,28 @@ export const InputFiles: InputFilesType = observer(
 
     const files = Array.from(value ?? []);
 
-    return styled(styles)(
-      <field className={s(sizeStyles, { ...layoutProps }, className)}>
-        <field-label title={labelTooltip || rest.title}>
+    return (
+      <div className={s(styles, { ...layoutProps, field: true }, className)}>
+        <label title={labelTooltip || rest.title} className={s(styles, { fieldLabel: true })}>
```

Comment:
Same with Label and Field, we can create separate components that will have styles in them